package-lock.json

```diff
@@ -0,0 +1,1897 @@
+{
```

Comment:
We are using yarn everywhere so I guess would be better to also use it here and not have the npm lock file here on top of the yarn lock